ansible/group_vars/all/defaults.yaml

```diff
@@ -43,7 +61,6 @@ kubernetes_cni_semver: v{{ kubernetes_cni_version }}
 kubernetes_cni_http_checksum: sha256:https://storage.googleapis.com/k8s-artifacts-cni/release/{{ kubernetes_cni_semver }}/cni-plugins-linux-amd64-{{ kubernetes_cni_semver }}.tgz.sha256
 kubernetes_cni_http_source: https://github.com/containernetworking/plugins/releases/download
 crictl_url: https://github.com/kubernetes-sigs/cri-tools/releases/download/v{{ crictl_version }}/crictl-v{{ crictl_version }}-linux-amd64.tar.gz
-crictl_sha256: 45e0556c42616af60ebe93bf4691056338b3ea0001c0201a6a8ff8b1dbc0652a
```

Comment:
nit: `crictl_sha256` used with `crictl_url` for Flatcar, can we keep these vars together please.

Reply:
I moved `crictl_sha256` because it must change whenever `crictl_version` changes. I'll think about this some more.

Reply:
Fixed